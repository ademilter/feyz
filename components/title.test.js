import { render, screen } from '@testing-library/react'
import Title from './title'

describe('Title', () => {
  test('should have children.', () => {
    const child = '<span>Test Message</span>'

    const { asFragment } = render(<Title>{child}</Title>)

    expect(asFragment()).toMatchSnapshot()
    expect(screen.getByText(/Test Message/i)).not.toBeNull()
  })
})
