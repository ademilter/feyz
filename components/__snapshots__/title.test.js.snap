// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A should have children. 1`] = `
<DocumentFragment>
  <h1
    class=""
  >
    <a
      href="/"
    >
      &lt;span&gt;Test Message&lt;/span&gt;
    </a>
  </h1>
</DocumentFragment>
`;
